src/utils.jl

```diff
@@ -242,3 +242,86 @@ flat_product(xs...) = vec(collect(Iterators.product(xs...)))
 Equivalent to `map(f, flat_product(xs...))`.
 """
 map_prod(f, xs...) = map(f, flat_product(xs...))
+
+"""
```

Comment:
Observe that most of the increase in the number of lines of code is associated to this function, `misty_closure`, and associated tests.